pages/dashboard/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Container, Typography } from '@mui/material';
+import { observer } from 'mobx-react';
+import { useRouter } from 'next/router';
+import { compose, JWTProps, jwtVerifier } from 'next-ssr-middleware';
+import { FC, useContext } from 'react';
+
+import { SessionBox } from '../../components/User/SessionBox';
+import { I18nContext } from '../../models/Translation';
+
+interface DashboardPageProps extends JWTProps {}
```

Comment:
```suggestion
interface DashboardPageProps extends JWTProps<User> {}
```
使用 `@idea2app/data-server` GitHub NPM 包中的类型